src/__testing__/__snapshots__/Header.test.js.snap

```diff
@@ -0,0 +1,118 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`snapshot testing snapshot for App 1`] = `
+<div>
+  <nav
+    className="nav_bar"
+  >
+    <a
+      href="/"
+      onClick={[Function]}
+    >
+      <svg
+        aria-hidden="true"
+        className="svg-inline--fa fa-circle-arrow-left "
+        data-icon="circle-arrow-left"
+        data-prefix="fas"
+        focusable="false"
+        id="arrow-left"
+        role="img"
+        style={Object {}}
+        viewBox="0 0 512 512"
+        xmlns="http://www.w3.org/2000/svg"
+      >
```

Comment:
- [ ] [OPTIONAL] Great job done @Becky449 , however, I will like to recommend that you remove the back button on the landing-page. You can just display it as none. This will enhance a better UI.

<img width="403" alt="Screenshot 2023-02-10 at 19 58 29" src="https://user-images.githubusercontent.com/58692590/218176452-b68186ee-647e-437c-b22f-f0ed908194c3.png">